infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/CommonPVCStrategy.java

```diff
@@ -199,8 +202,10 @@ public void prepare(KubernetesEnvironment k8sEnv, String workspaceId, long timeo
     if (!existing.contains(commonPVC.getMetadata().getName())) {
       log.debug("Creating PVC for workspace '{}'", workspaceId);
       pvcs.create(commonPVC);
-      log.debug("Waiting PVC for workspace '{}' to be bound", workspaceId);
-      pvcs.waitBound(commonPVC.getMetadata().getName(), timeoutMillis);
+      if (waitBound) {
+        log.debug("Waiting PVC for workspace '{}' to be bound", workspaceId);
```

Comment:
Proper English here would be something like "Waiting for workspace PVC {} to be bound"

Reply:
I guess this should be changed to `Waiting for PVC(s) of workspace {} to be bound.` as in the other comment.